Description: Fix linker flag on kfreebsd-* ports
 GNU/kFreeBSD is detected as Linux (see CONFIG/src/probe_OS.c), hence the test
 of the OS variable does not work.
Forwarded: not-needed
Last-Update: 2017-08-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -794,8 +794,10 @@ int main(int nargs, char **args)
             fprintf(fpout, " -melf_i386");
          else if (ptrbits == 64)
             fprintf(fpout, " -melf_x86_64");
-         if (OS == OSFreeBSD)
-            fprintf(fpout, "_fbsd");
+ #ifdef __FreeBSD_kernel__
+          fprintf(fpout, "_fbsd");
+#endif
+
       }
    }
    if (MachIsS390(mach))
